src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/DevAppServerArgs.java

```diff
@@ -24,30 +24,42 @@
 public class DevAppServerArgs {
 
   /**
-   * @return [--name=value] or [] if value=null.
+   * @return {@code [--name=value]} or {@code []} if value=null.
    */
   public static List<String> get(String name, String value) {
     return Args.stringWithEq(name, value);
   }
 
   /**
-   * @return [--name=value1, --name=value2, ...] or [] if value=null.
+   * @return {@code [--name=value1, --name=value2, ...]} or {@code []} if value=null.
    */
   public static List<String> get(String name, List<String> values) {
     return Args.stringsWithEq(name, values);
   }
 
   /**
-   * @return [--name=value] or [] if value=null.
+   * @return {@code [--name=value]} or {@code []} if value=null.
    */
   public static List<String> get(String name, Integer value) {
     return Args.integerWithEq(name, value);
   }
 
   /**
-   * @return [--name] if value=true, [] if value=false/null.
+   * @return {@code [--name]} if value=true, {@code []} if value=false/null.
    */
   public static List<String> get(String name, Boolean value) {
+    return get(name, value, false);
+  }
+
+  /**
+   * @return if produceTrueFalse=true, {@code [--name=true]} if value=true, {@code [--name=false]}
```

Comment:
@joaoandremartins but the issue with that is that defaults are not always 'false'. Which means the caller has to know which one of these methods to call based on the defaults in gcloud.  That seems like the wrong thing to do.